assets/sass/scaffolding/grid.scss

```diff
@@ -212,3 +212,43 @@
         margin: 0 !important;
     }
 }
+
+$num_columns: 12;
+$num_columns_mobile: 3;
+
+@mixin generate-column($className, $width) {
+    #{$className} {
+        flex-basis: $width;
+        flex-grow: 0;
+
+        .grid--padded & {
+            flex-basis: calc(#{$width} - #{$paddedGridGutter});
+        }
+
+        .grid--padded--large & {
+            flex-basis: calc(#{$width} - #{$paddedGridGutterLarge});
+        }
+    }
+}
+
+.row {
+    flex-wrap: wrap;
+}
+
+@include mq('tablet', 'max') {
+    @for $i from 1 through $num_columns_mobile {
+        $width: $i / $num_columns_mobile * 100%;
+        .grid__item {
+            @include generate-column('&.col-#{$i}--m', $width);
```

Comment:
🤓  Pedantic, but maybe `.grid-col--` to make it explicit it's part of the grid classes?